### ISSUE TITLE
Strange, contradictory, output of linter

### ISSUE THREAD

**Issue by Defossez** [AtomLinter/linter-markdown#154]
Hi,

Using Atom 1.22.1 x64, installed linter-markdown 5.2.0.
I'm experimenting with markdown and installed the linter, but that provides strange warnings over a markdown file loaded in Atom. I'm using a ReadMe.markdown file from NVM to test and see markdown stuff. The file is open in Atom and markdown-preview-enhanced nicely displays the document.
But there are many red bullets in the left gutter. All bullets from linter-markdown.
One of the warnings is:
      Incorrect indentation before bullet: remove 2 spaces (remark-lint:list-item-bullet-indent)
Could be a fair point, so I removed the two spaces.
Then the linter produces this warning:
      Incorrect list-item indent: add 2 spaces (remark-lint:list-item-indent)

Spoken about mixed messages ....
Why is the linter doing this?
     Probably some setting in one or another file (remarkrc)?

If it is that file, how does it need to look?
    I've tried several things but the above sort of messages keep popping up.

Thanks,

Marc

**Comment by oupala** [comment]
I have the same stand contradictory messages.

Here is the `.remarkrc` file I'm using:

```json
{
  "plugins": [
    "remark-preset-lint-recommended",
    "remark-preset-lint-consistent",
    ["remark-lint-list-item-indent", "space"],
    ["remark-lint-table-cell-padding", "padded"]
  ]
}
```

Any clue on how to make this behavior more consistent?

**Comment by wooorm** [comment]
@Defossez Sorry for the late response.

You can find more about these messages on remarkjs/remark-lint. Such as [list-item-bullet-indent](https://github.com/remarkjs/remark-lint/tree/master/packages/remark-lint-list-item-bullet-indent) and [remark-lint-list-item-indent](https://github.com/remarkjs/remark-lint/tree/master/packages/remark-lint-list-item-indent).

@oupala If this is related to a globally installed stuff, then it is related to GH-156.
